<launch>
<node name="Mapping_Node" pkg="icarus_rover_rc" type="Mapping_Node" output="screen" >
		  <param name="Operation_Mode" value="SIM"/>
		<param name="DEBUG_MODE" value="1"/>  
		<param name="LOGGING_ENABLED" value="0"/>
		<param name="Beams_Per_Sonar" value="10"/>
		<param name="Sonar_Beamwidth" value="30"/>
		<param name="Sonar_MaxDistance" value = "6.5024"/>
		
	</node>
</launch>
